iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -155,7 +155,13 @@ export class Inspector {
 
     handleMessage(event) {
         const jsonString = event.originalEvent.data;
-        const data = JSON.parse(jsonString);
+        let data;
+        try {
+            data = JSON.parse(jsonString);
+        }
+        catch (e) {
+            return;
```

Comment:
It would be good to add a comment here explaining that we're swallowing exceptions in order to ignore messages from the `write-excel-file` library